src/misclibs.F

```diff
@@ -3205,7 +3205,7 @@ subroutine calccflquick(dt,uh,vh,mh,ua,va,wa,reqc)
       dtdy=0.5*dt*rdy
       dtdz=0.5*dt*rdz
 
-      !$acc parallel private(i,j,k,wsp) reduction(max:fmax)
+      !$acc parallel default(present) private(i,j,k,wsp) reduction(max:fmax)
```

Comment:
maybe remove `private` clause?

Reply:
maybe remove `private` clause for the `acc loop` directive and add `gang vector` clause there?